### PR TITLE
CI-CD-production-android

### PR DESCRIPTION
**Pull request by Mnoble-19**
#### Summary of Changes (What does this PR do?)

- Refactor firebase app distribution worflow (rename gcloud secrets appropriately - cc @123MwanjeMike )
- Set up workflow for pushing android to production

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [x] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Trigger workflow from Github Actions sections.

#### What are the relevant tickets?
- [MOB-21](Ihttps://airqoteam.atlassian.net/browse/MOB-21?atlOrigin=eyJpIjoiYmFmMTM2NjM3YzkyNDY4ZmI4MzVjZjcwNzA2ZjcxNGEiLCJwIjoiaiJ9)

[MOB-21]: https://airqoteam.atlassian.net/browse/MOB-21?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/deploy-android-to-firebase-app-distribution.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    inputs:
+      android:
+        description: "Deploy android to firebase app distribution"
+        required: false
+        type: boolean  
+        
+jobs:
+  android:
+    name: android build
+    runs-on: ubuntu-latest
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-android-to-firebase-app-distribution.yml:14]: Could we add the line  `if: inputs.android == true` here just below the the job name. This tells the job to run when the box is checked

File: .github/workflows/deploy-android-to-play-console.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    inputs:
+      android:
+        description: "Deploy android to play store"
+        required: false
+        type: boolean  
+        
+jobs:
+  android:
+    name: android build
+    runs-on: ubuntu-latest
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-android-to-play-console.yml:14]: Same here

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]
This is great work @Mnoble-19 
Just a one small thing missing

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @Mnoble-19 
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**